omero/developers/installation.txt

```diff
@@ -53,38 +53,8 @@ Building OMERO
 
 To install the dependencies required to run the OMERO.server on Linux
 or Mac OS X, take a look at the
-:doc:`/sysadmins/unix/server-installation` or the
-:doc:`/sysadmins/unix/server-install-homebrew` sections.
-
-Additional dependencies are necessary for building the server and clients:
-
-- Ice Python and Java development packages may need to be explicitly installed
-  on Linux.
-
-Finally, some environment variables may need to be set up before building
-the server:
-
-- If the system slice files cannot be found you must set :envvar:`SLICEPATH`
-  to point to the :file:`slice` directory of the Ice installation.
```

Comment:
I remember setting this environment variable is specific to building the server from the source code.